cmd/export-service/api_server.go

```diff
@@ -49,7 +49,8 @@ func createPublicServer(cfg *config.ExportConfig, external exports.Export) *http
 	)
 
 	router.Get("/", statusOK)
-	router.Get("/api/export/v1/openapi.json", serveOpenAPISpec(cfg)) // OpenAPI Spec
+	router.Get("/api/export/v1/public.json", servePublicOpenAPISpec(cfg)) // OpenAPI Specs
```

Comment:
I think this should be exported as `openapi.json` instead of `public.json` based on [this](https://consoledot.pages.redhat.com/docs/dev/developer-references/rest/requirements.html)